docs/antora/modules/ROOT/pages/configuration/index.adoc

```diff
@@ -26,25 +34,20 @@ In this section, we discuss the common and specific configurations of all the mi
 === Common configurations
 The common configurations described in this section apply to all the microservices within Lisk Service.
 
-[tabs]
-=====
-Properties::
-+
---
-[cols="2,1,3,3",options="header",stripes="hover"]
+[cols="2,1,~,~m",options="header",stripes="hover"]
 |===
 |Environment variable
 |Datatype
 |Description
 |Sample
 
-| `SERVICE_BROKER`
-| string
-| URL of the microservice message broker.
+|`SERVICE_BROKER`
+|string
+|URL of the microservice message broker.
 *Must be identical for all the microservices.*
 Make sure that all microservices can connect with the message broker.
 This could be a NATS or a Redis instance.
-| redis://localhost:6379/0
+|redis://localhost:6379/0
```

Comment:
Can we please update all instances of URLs containing `localhost` to `127.0.0.1` instead? This could cause issues to users on MacOS Ventura without proper network configuration (`/etc/hosts`).
```suggestion
|redis://127.0.0.1:6379/0
```